react/features/notifications/components/web/NotificationsContainer.js

```diff
@@ -290,14 +289,15 @@ class NotificationsContainer extends Component<Props> {
 function _mapStateToProps(state) {
     const { notifications } = state['features/notifications'];
     const { iAmSipGateway } = state['features/base/config'];
+    const { notificationTimeouts } = state['features/base/config'];
     const { isOpen: isChatOpen } = state['features/chat'];
     const _visible = areThereNotifications(state);
 
     return {
         _iAmSipGateway: Boolean(iAmSipGateway),
         _isChatOpen: isChatOpen,
         _notifications: _visible ? notifications : [],
-        autoDismissTimeout: interfaceConfig.ENFORCE_NOTIFICATION_AUTO_DISMISS_TIMEOUT
+        autoDismissTimeout: notificationTimeouts?.long ?? NOTIFICATION_TIMEOUT.LONG
```

Comment:
Get rid of this altogether, the notification will now always have a timeout set.